sanitizers/src/test/java/com/example/BUILD.bazel

```diff
@@ -161,6 +161,19 @@ java_fuzz_target_test(
     verify_crash_reproducer = False,
 )
 
+# Due to system dependant behavior of JVMs this test is set to manual execution.
```

Comment:
Could we expand this comment a bit more? We should at least say that catching `StackOverflowError` doesn't work reliably since it may have lead to a native crash before we can handle it in Java.

Reply:
Right, I extended the comment. Is it better now?